tests/conftest.py

```diff
@@ -1,6 +1,9 @@
 import os
+import tempfile
 from . import DATA_DIR
 
 
-# Setting the environment variable to the config file path before importing nplinker in any test
+# Specify the config file via environment variable before importing nplinker in any test.
 os.environ["NPLINKER_CONFIG_FILE"] = str(DATA_DIR / "nplinker_demo1.toml")
```

Comment:
Why don't we have a test for "nplinker_default.toml" as well?

Reply:
The `nplinker_default.toml` is always pre-loaded (see `config = Dynaconf(settings_files=[user_config_file], preload=[default_config_file])` in the `config.py` file).
New values give by users will overwrite the default values.

Reply:
Why we have to specify the env variable in the `tests/conftest.py` file?
The execution of pytest starts from this file, so providing the env varibles for config before the config is imported (i.e. the config.py is triggered) by any code.